changelog.json

```
@@ -1,5 +1,10 @@
 {
   "releases": {
+    "2.9.7-beta2": [
+      "[New] Initial support for system notifications when checks fail in Windows - #13655",
+      "[Improved] Upgrade to Electron v16.0.8 - #13814",
+      "[Improved] Use perl to resolve symlinks on macOS - #13764. Thanks @spotlightishere!"
```

I think I'd be more generic with this:
```suggestion
      "[Fixed] Fix CLI tool on macOS Monterey - #13764. Thanks @spotlightishere!"
```

---

You might need to move lines around (Fixed lines are above Improved)